server/controllers/BookController.js

```diff
@@ -75,6 +75,70 @@ class BookController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+  /**
+* Edit Books
+* @name editBook
+* @param {Object} request
+* @param {Object} response
+* @returns {JSON} object
+* @memberof BookController
+*/
+  static async editBook(request, response) {
+    const { id } = request.params;
+    const {
+      title, description, isbn, price, yearPublished, authorId
+    } = request.body;
+    // const bodyData = title || description || isbn || price || yearPublished;
```

Comment:
remove this commented code from here since you are not making use of it.

Reply:
this is not commented .... have no idea why you have it commented, will recommit and push again